Sources/Logging/Loggers/ModuleLogger.swift

```diff
@@ -27,6 +27,7 @@ public protocol ModuleLogger: Logger {
     ///   - file: The file from where the log was invoked.
     ///   - line: The line from where the log was invoked.
     ///   - function: The function from where the log was invoked.
+    // swiftlint:disable:next function_parameter_count
```

Comment:
I believe this will mess with the documentation above it. Does Alt-clicking still work fine?

Reply:
> doesn't it work if we put it before the documentation?

That was my first attempt. The warning persisted. But I found a way to not use the `previous` here. I used `this`. I used the `disable:this` right next to the first parameter declaration. Since all parameters are "aligned" it goes around this issue a bit.

It not looks like this:

```
    /// - Parameters:
    ///   - module: The module from which the message originated.
    ///   - level: The severity level of the message.
    ///   - message: The log message.
    ///   - file: The file from where the log was invoked.
    ///   - line: The line from where the log was invoked.
    ///   - function: The function from where the log was invoked.
    func log(module: Module, // swiftlint:disable:this function_parameter_count
             level: Log.Level,
             message: @autoclosure () -> String,
             file: StaticString,
             line: UInt,
             function: StaticString)
```

Not "perfect", it's the best I can do with the current state of the tools